sensing/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -135,18 +141,33 @@ void GNSSPoser::callbackNavSatFix(
     canGetCovariance(*nav_sat_fix_msg_ptr) ? nav_sat_fix_msg_ptr->position_covariance[4] : 10.0;
   gnss_base_pose_cov_msg.pose.covariance[6 * 2 + 2] =
     canGetCovariance(*nav_sat_fix_msg_ptr) ? nav_sat_fix_msg_ptr->position_covariance[8] : 10.0;
-  gnss_base_pose_cov_msg.pose.covariance[6 * 3 + 3] = 0.1;
-  gnss_base_pose_cov_msg.pose.covariance[6 * 4 + 4] = 0.1;
-  gnss_base_pose_cov_msg.pose.covariance[6 * 5 + 5] = 1.0;
+  gnss_base_pose_cov_msg.pose.covariance[6 * 3 + 3] =
+    std::pow(gnssOrientationRmse.rmse_rotation_x * deg2rad, 2);
```

Comment:
If the "/autoware_orientation" topic fails to subscribe, I want the covariance to set a default value so that it does not become 0.

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1230/commits/e7757d4f7d890ba721436139b4cc32ca82f77d81